buildenv/jenkins/testJobTemplate

```diff
@@ -173,8 +173,9 @@ ARCH_OS_LIST.each { ARCH_OS ->
 				}
 
 				// for jck builds, if JCK_GIT_REPO_PREFIX is set, set JCK_GIT_REPO if it is not set
+				def ACTUAL_JCK_GIT_REPO = ""
 				if (GROUP == "jck" && JCK_GIT_REPO_PREFIX && !JCK_GIT_REPO) {
-					JCK_GIT_REPO = "${JCK_GIT_REPO_PREFIX}/JCK${JDK_VERSION}-unzipped.git"
+					ACTUAL_JCK_GIT_REPO = "${JCK_GIT_REPO_PREFIX}/JCK${JDK_VERSION}-unzipped.git"
```

Comment:
I think the original code supports two cases:
- the user provides `JCK_GIT_REPO` repo. The script uses it to generate test jobs
- the user does not provide `JCK_GIT_REPO` repo. The script sets `JCK_GIT_REPO` to `${JCK_GIT_REPO_PREFIX}/JCK${JDK_VERSION}-unzipped.git`   // this is ideal for running in a loop

Also, the change in this PR will cause user-provided `JCK_GIT_REPO` will be ignored.

Reply:
Good catch. I think I've fixed it now.